doc/source/cluster/autoscaling.rst

```diff
@@ -112,37 +112,28 @@ The above config defines two CPU node types (``cpu_4_ondemand`` and ``cpu_16_spo
 
 The node config tells the underlying Cloud provider how to launch a node of this type. This node config is merged with the top level node config of the YAML and can override fields (i.e., to specify the p2.xlarge instance type here):
 
-.. code::
+.. code-block:: yaml
 
     node_config:
         InstanceType: p2.xlarge
 
 The resources field tells the autoscaler what kinds of resources this node provides. This can include custom resources as well (e.g., "Custom2"). This field enables the autoscaler to automatically select the right kind of nodes to launch given the resource demands of the application. The resources specified here will be automatically passed to the ``ray start`` command for the node via an environment variable. For more information, see also the `resource demand scheduler <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/resource_demand_scheduler.py>`__:
 
-.. code::
+.. code-block:: yaml
 
     resources: {"CPU": 4, "GPU": 1, "Custom2": 2}
 
 The ``min_workers`` and ``max_workers`` fields constrain the minimum and maximum number of nodes of this type to launch, respectively:
 
-.. code::
+.. code-block:: yaml
 
     min_workers: 1
     max_workers: 4
 
 The ``worker_setup_commands`` field (and also the ``initialization_commands`` field, not shown) can be used to override the setup and initialization commands for a node type. Note that you can only override the setup for worker nodes. The head node's setup commands are always configured via the top level field in the cluster YAML:
 
-.. code::
+.. code-block:: yaml
 
     worker_setup_commands:
         - pip install tensorflow-gpu  # Example command.
 
-Docker Support
-~~~~~~~~~~~~~~
-The ``worker_image`` and ``pull_before_run`` fields override the correpsonding field in the top level ``docker`` section for the node type. The ``worker_run_options`` field is combined with top level ``docker: run_options`` field to produce the docker run command for the given node_type. The following configuration is for a GPU enabled node type. Ray will automatically select the Nvidia docker runtime if it is available.
-
-.. code::
-
-    pull_before_run: True
-    worker_image:
-        - rayproject/ray-ml:latest-gpu
```

Comment:
Can you keep this section here and rename it to `Docker Support in Multi-Node Clusters` or something similar...That is what this is describing

Reply:
done, good catch.